network/datadog_checks/network/network.py

```diff
@@ -270,7 +270,13 @@ def _is_collect_cx_state_runnable(self, proc_location):
             return False
```

Comment:
Some level of coverage for this change can be helpful (`test_network.py`). We might need to mock things, but will still help.

Reply:
https://github.com/DataDog/integrations-core/pull/7095/commits/7271fbcaf6ced27c5f6e0394188e9c0ba841470d

Reply:
@L3n41c Thx for this test.

Sorry for not being clear upfront. I was initially referring to a test that verifies that `subprocess_output` is receiving the correct `env`. Can be done using mocks and `assert_called_with` on `subprocess_output` mock.

Reply:
Hi @AlexandreYang,
Here is a test that checks that `subprocess_output` is getting the proper `env`: https://github.com/DataDog/integrations-core/pull/7095/commits/22907724e050826fcce2a2627dca9334443e6972

Reply:
Thx 👍